src/components/Books.js

```diff
@@ -18,9 +18,8 @@ const Books = () => {
   ));
 
   return (
-    <div>
-      <h1>Books</h1>
-      <ul>
+    <div className="container">
+      <ul className="book-list" style={{ display: 'flex', flexDirection: 'column', gap: '20px' }}>
         {bookList}
       </ul>
```

Comment:
- [ ] In terms of the styling, it could be improved too 👷🏼.  Please consider using all the available space moving the progress information all the way to the right! 🤓 🔧 

- [ ] Also, the section has a light gray background, and each card has a border and a white background that differentiates them from the background. Please consider applying this style! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/215219495-a9c80e23-7fb6-415d-9a77-6f785962ad91.png) | ![image](https://user-images.githubusercontent.com/77069011/215219413-1cbd8fe7-8200-410e-9c5e-5aa937a95d4c.png) |